ui/components-react/src/components-react/filter-builder/FilterBuilderRule.scss

```diff
@@ -14,6 +14,7 @@
     grid-column-gap: var(--iui-size-m);
     width: calc(100% - var(--iui-size-s));
 
+    // TODO: iTwinUI 3.x
     .iui-input-container.iui-with-message {
```

Comment:
@saskliutas would you mind helping me to replace these .iui class names? I had trouble finding out how to simulate rule error messages in our storybook.

Reply:
Removed in https://github.com/iTwin/appui/pull/666/commits/90de4a468248c7ab6e315a1e0348495f18765c23